tests/stdlib/fixtures/simple_dump_to_zip_with_hash_&_pretty_descriptor

```diff
@@ -0,0 +1,60 @@
+dump.to_zip
```

Comment:
where are you testing the actual change in behaviour?

Reply:
@akariv This test is indirect - it checks if the output zip has the same hash and size, as a sample with pretty descriptor, that I've got when implemented the feature.

This is not the best way to test this feature, but I don't know how to integrate a proper test in existing repo test system, that uses 'fixtures' :(

Other 'dump' tests (dump.to_sql, dump.to_zip) follows the same way, so did I.

Just looked into `test_main.py` - and it checks if pipeline test was successful in the same way - by checking output bytes and hash is the same as in the sample.

If you know the better way to create this test, please tell me.

Reply:
Okay , got it, np